docs/source/conf.py

```diff
@@ -1,14 +1,17 @@
 from datetime import datetime
 
-# Adding BentoML source directory for accessing BentoML version
-import bentoml
+try:
```

Comment:
@aarnphm same here. let's revert them for now and submit a separate PR for review